src/test/kotlin/uk/gov/justice/digital/hmpps/externalmovementsapi/integration/sync/SynTapApplicationIntTest.kt

```diff
@@ -57,7 +58,7 @@ class SynTapApplicationIntTest(
     val prisonCode = "TAU"
     val legacyId = newId()
     val existing = givenTemporaryAbsenceSeries(temporaryAbsenceSeries(prisonCode, legacyId = legacyId))
-    val request = applicationRequest(prisonCode, movementApplicationId = legacyId)
+    val request = applicationRequest(prisonCode, movementApplicationId = legacyId, id = existing.id)
```

Comment:
I think a third test covering application created successfully with supplied id would be helpful